### DISCUSSION TITLE
Qestion using GFXFF Fonts

### DISCUSSION THREAD

**Discussion by hasenradball** [Bodmer/TFT_eSPI#1742] [Q&A - Fonts] [answered]
Hi,

if I want to use the gfxff fonts, is there a trick to include the specific header?
In your example you created a specifc "Font.h" file.

**Comment by Bodmer** [comment, marked as answer]
If the following is defined in your library setup file then a sketch will have access to the GFX free fonts:
#define LOAD_GFXFF

There are examples for using the fonts. Some examples include a header file that provides shorter reference names, but that is not necessary to reference a font by the full name.
https://github.com/Bodmer/TFT_eSPI/blob/master/examples/320%20x%20240/All_Free_Fonts_Demo/Free_Fonts.h

**Comment by hasenradball** [comment]
Hi,
with the lastest release I see some issue when drawing a float, see here(sorrx for the bad quality):
![grafik](https://user-images.githubusercontent.com/26808969/161431804-132b33ca-8180-4cc5-9101-4b223fd41168.png)

I drawed this like:

```
void TFTweather::showTemperature(float temperatur, const cursor_pos &pos, uint16_t color_txt, uint16_t color_bg) {
  tft.setTextDatum(TR_DATUM);
  tft.setFreeFont(FSS24);
  int padding = tft.textWidth("-00.0", GFXFF);
  tft.setTextColor(color_txt, color_bg);
  tft.setTextPadding(padding);
  tft.drawFloat(temperatur, 1, pos.x + padding, pos.y, GFXFF);
}
```
Do I anything wrong?

**Reply by hasenradball** [reply]
@Bodmer can you have a short look to this issue?

**Reply by Bodmer** [reply]
The code looks fine, however it does not use an oblique font, so maybe there is a problem in the library for oblique free fonts? I will investigate but it will be a while before I can do that.
